Resolve clang-tidy warning about end of non-void functions

mnurzyns opened the pull request:
All functions which should return non-void value and have a chance
to do not do it was change. Most of them can not reach if...else
but some of them call other fuction which is void.

mnurzyns committed:
Resolve clang-tidy warning about end of non-void functions

All functions which should return non-void value and have a chance
to do not do it was change. Most of them can not reach if...else
but some of them call other fuction which is void.

NMarcin commented:
Check changes by file.

marcinwozniak0 commented on MessengerOnFileForLinux/FileHandling/src/FileHandling.cpp:
```diff
@@ -356,6 +356,7 @@ bool FileInterface::Modification::updateRowField(const std::string& pathToFile,
 bool FileInterface::lockFolder(const std::string& pathToFolder)
 {
     waitForAccess(pathToFolder);
+    return not FileInterface::Managment::isFileExist(pathToFolder + "/GUARD");
```
1. FileInterface:: niepotrzebne. 
2. A może lockFolder zmienić na void, i tak nie korzystamy nigdzie z żadnego przypisania wyniku tej funkcji ?
3. Dodatkowo jak już tu jesteś, to przydałoby się zabezpieczyć funckję przed sytuacją kiedy nie udało nam się stowrzyć guarda. Bo aktualnie w waitForAccess() wywołujemy createGuard() i radośnie zakładamy że się zawsze utworzy.
4. Trzeba zrobić taska na refaktor tego pliku, bo aż mi sie niedobrze robi jak patrze na funckje wait..., która robi jeszcze w srodku milion rzeczy :-D

mnurzyns replied:
1. Tak, ale trzymałem się konwencji z Twojej funkcji na dole.
2. nie void, bo wtedy nie kontrolujemy potem. Wiem, że nie korzystamy z żadnego rpzypisania bo grepowałem, ale uznałem, że wolimy mieć funkcję gotową na korzystanie z tego niż potem to jeszcze raz zmieniać.
3. To nie jest scope tego taska. Tworzę nowego taska na to.
4. Nie chcę sugerować kto to pisał :package:

mnurzyns replied:
A tak wgl to jest błąd, bo nie powinno być "not" w funkcji

marcinwozniak0 commented on MessengerOnFileForLinux/MessageFramework/src/Message.cpp:
```diff
@@ -53,6 +53,11 @@ bool Message::setMessageFlag(std::string messageFlag)
     }
+    else
+    {
+        log_.info("Message::setMessageFlag() messageFlag unknown, error");
```
nie mieliśmy konwencji Message::setMessageFlag() ERROR: messageFlag unknown ?

mnurzyns replied:
możliwe, że mieliśmy (nie kojarzę nadal), ale i tak będzie wypadała, bo w nowym Loggerze będzie to jako parametr STATUS. INFO, DEBUG, ERROR itd.

mnurzyns committed:
Aligment misteake in checking is guard created

When we want to check is guard was creatad we should to check is
file named GUARD exist.
